pkgs/tools/networking/telepresence2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "telepresence2";
+  version = "2.1.4";
+
+  src = fetchFromGitHub {
+    owner = "telepresenceio";
+    repo = "telepresence";
+    rev = "v${version}";
+    sha256 = "0y7np7i7isj0ydli3n33hcw8id24zvnkn4ghih917r2wd74r44f4";
+  };
+
+  vendorSha256 = "0s2fyb77zkr97dknsay7sz9xhm59wmga89dipcg7d17v0q5inz9m";
+
+  buildFlagsArray = ''
+    -ldflags=-X=github.com/telepresenceio/telepresence/v2/pkg/version.Version=${src.rev}
```

Comment:
```suggestion
    -ldflags=-s -w -X=github.com/telepresenceio/telepresence/v2/pkg/version.Version=${src.rev}
```

Reply:
Applied in c03d2caa859bb90816ba1d32b61e567fcf98da98